libs/utils/utils.js

```diff
@@ -1230,11 +1230,12 @@ export function partition(arr, fn) {
   );
 }
 
-const preloadBlockResources = (blocks = []) => blocks.map((block) => {
+export const preloadBlockResources = (blocks = []) => blocks.map((block) => {
   if (block.classList.contains('hide-block')) return null;
   const { blockPath, hasStyles, name } = getBlockData(block);
-  if (['marquee', 'hero-marquee'].includes(name)) {
+  if (['marquee', 'hero-marquee', 'homepage-brick'].includes(name)) {
     loadLink(`${getConfig().base}/utils/decorate.js`, { rel: 'preload', as: 'script', crossorigin: 'anonymous' });
+    if (name === 'homepage-brick') loadLink(`${getConfig().base}/martech/attributes.js`, { rel: 'preload', as: 'script', crossorigin: 'anonymous' });
```

Comment:
Is the homepgae-brick part of a test? I'd try not to add new code for perf reasons while a test is running. We should keep utils slim and optimize for the >90% cases